po/vi.po

```diff
@@ -5631,26 +5613,24 @@ msgstr ""
 msgid ""
 "The contents are treated as Markdown. All published Rust library crates are "
 "automatically documented at [`docs.rs`](https://docs.rs) using the [rustdoc]"
-"(https://doc.rust-lang.org/rustdoc/what-is-rustdoc.html) tool. It is "
-"idiomatic to document all public items in an API using this pattern."
```

Comment:
The diff is including the English portions as well. This shouldn't be the case. Is this caused by the formatter?